scenarios/aws/microVMs/microvms/provision.go

```diff
@@ -27,9 +28,38 @@ func GetSudoPassword(ctx *pulumi.Context, isLocal bool) pulumi.StringOutput {
 	return SudoPasswordRemote
 }
 
+func sharedFSMountpoint() string {
+	return filepath.Join(GetWorkingDirectory(), "kmt-ramfs")
+}
+
+const createRAMFS = `mkdir -p %[1]s && \
+sudo mount -t tmpfs -o size=%[2]s tmpfs %[1]s \
+`
+
+const exportRAMFS = `sudo sh -c 'echo "%[1]s 100.0.0.0/8(ro,no_root_squash,no_subtree_check)" >> /etc/exports && exportfs -a '`
+
 // This function provisions the metal instance for setting up libvirt based micro-vms.
-func provisionInstance(_ *Instance) ([]pulumi.Resource, error) {
-	return []pulumi.Resource{}, nil
+func provisionInstance(instance *Instance) ([]pulumi.Resource, error) {
+	runner := instance.runner
+
+	createRAMFSArgs := command.Args{
+		Create: pulumi.Sprintf(createRAMFS, sharedFSMountpoint(), "20G"),
+		Delete: pulumi.Sprintf("umount %s", sharedFSMountpoint()),
+	}
+	createRAMFSDone, err := runner.Command(instance.instanceNamer.ResourceName("create-ram-fs"), &createRAMFSArgs)
+	if err != nil {
+		return []pulumi.Resource{}, err
```

Comment:
`nil` is equivalent to an empty slice in Go
```suggestion
		return nil, err
```